diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/comments/HeaderCommentRule.kt

```diff
@@ -190,6 +190,25 @@ class HeaderCommentRule(private val configRules: List<RulesConfig>) : Rule("head
         }
     }
 
+    private fun handleMultilineCopyright(copyrightText: String): String {
+        var properCopyright = ""
+        if (copyrightText.contains("\n")) {
+            copyrightText.split("\n").map { "    $it" }.forEach {
+                properCopyright += if (it == "    ") {
```

Comment:
Also I'm sure there is a method in kotlin stdlib to prepend symbols for all lines. Or at least it can be done via `runningFold` or `reduce`, without mutable variables.